test/unit_tests/braket/circuits/test_noise.py

```diff
@@ -387,41 +387,6 @@ def __init__(self):
     assert Noise._FooNoise().name == _FooNoise().name
 
 
-@pytest.mark.parametrize(
```

Comment:
Can you please move this test to the test_noises function and make sure all the bind_values work for all the noises?

Reply:
Why test_noises? This is only testing the superclasses, not the implementations.

Reply:
It's actually [already in the noise_models branch](https://github.com/aws/amazon-braket-sdk-python/blame/param_noise/test/unit_tests/braket/circuits/test_noises.py#L481), which is why it isn't showing up here.